projects/driver_controls/src/ads1015.c

```diff
@@ -0,0 +1,151 @@
+#include <ads1015.h>
+#include <stdbool.h>
+#include <stdio.h>
+
+#include "ads1015_bitmasks.h"
+#include "delay.h"
+#include "gpio_it.h"
+#include "i2c.h"
+
+// Union definition to help process registers data
+typedef struct ADS1015Register {
+  uint8_t data[2];
```

Comment:
The size of this array is used everywhere I recommend making the `2` a macro.

Reply:
Since the same amount of bytes are exchanged per read and write, I got rid of the len parameter. This should remove the need for the magic byte lengths in the program